package.json

```diff
@@ -282,6 +282,7 @@
     "@metamask-institutional/sdk": "^0.1.27",
     "@metamask-institutional/transaction-update": "^0.2.2",
     "@metamask/abi-utils": "^2.0.2",
+    "@metamask/account-watcher": "^3.1.1",
```

Comment:
Since it's pre-installed, do we still need it here?

Reply:
Yes cause we need to grab the `preinstalled-snap.json` from the npm package [here](https://github.com/MetaMask/metamask-extension/blob/feat/preinstalled-snap-confirmations-bypass/app/scripts/snaps/preinstalled-snaps.ts#L3) :)